content/src/main/content/jcr_root/apps/dam/gui/content/assets/metadataeditor/.content.xml

```diff
@@ -0,0 +1,85 @@
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="acs-commons/renderconditons/ispublished"
+                                        resourcePath="${paramValues['item'][0]}"/>
+                            </granite:rendercondition>
+
+                        </copypublishurl>
+                    </items>
+                </col1>
+                <dialogpublishurl
+                    granite:id="aem-assets-show-publish-url"
+                    jcr:primaryType="nt:unstructured"
+                    jcr:title="Publish URL"
+                    sling:resourceType="granite/ui/components/coral/foundation/dialog"
+                    closable="{Boolean}true">
+                    <granite:data
+                        jcr:primaryType="nt:unstructured"
+                        assetpath="${empty requestPathInfo.suffix ? paramValues[&quot;item&quot;][0] : requestPathInfo.suffix}"/>
+                    <items jcr:primaryType="nt:unstructured">
+                        <textarea
+                            granite:rel="asset-publishurl-text-content"
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="granite/ui/components/coral/foundation/form/textarea"/>
+                    </items>
+                    <footer jcr:primaryType="nt:unstructured">
+                        <copytext
+                            granite:id="asset-publishurl-copy-cmd"
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="granite/ui/components/coral/foundation/button"
+                            icon="attach"
+                            iconSize="XS"
+                            text="Copy Publish URL to clipboard"
+                            variant="minimal">
+                            <granite:rendercondition
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/renderconditions/simple"
+                                expression="${!granite:containsIgnoreCase(header[&quot;User-Agent&quot;], &quot;iPad&quot;)}"/>
+                        </copytext>
+                        <dismiss
+                            granite:rel="foundation-toggleable"
+                            jcr:primaryType="nt:unstructured"
+                            sling:resourceType="granite/ui/components/coral/foundation/button"
+                            text="Dismiss"
+                            variant="primary">
+                            <parentConfig
+                                jcr:primaryType="nt:unstructured"
+                                close="{Boolean}true"/>
+                        </dismiss>
+                    </footer>
+                </dialogpublishurl>
+            </items>
+        </content>
+        <head jcr:primaryType="nt:unstructured">
```

Comment:
does this really need to be here? again, this potentially conflicts with other customizations.